packages/auth/src/interaction/routes.ts

```diff
@@ -181,7 +181,6 @@ async function startInteraction(
 
   const trx = await Interaction.startTransaction()
   try {
-    // TODO: also establish session in redis with short expiry
```

Comment:
confirmed with Nathan this was just about moving sessions into Redis so its no longer a TODO